.aio

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
please gitignore .aio file, as it contains project specific details.

Reply:
Done. Thank you 😄